### PR TITLE
Issue 1431: fix copy multiple locators + small refactoring

### PR DESCRIPTION
**Pull request by MariiaNebesnova**

### CODE REVIEW

File: src/features/locators/utils/copyLocatorToClipboard.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    switch (match) {
+      case "\\":
+        return "\\\\\\\\";
+      case "'":
+        return "\\'";
+      case "\n":
+        return "\\n";
+      default:
+        return match;
+    }
```
Comment (Iogsotot) [inline src/features/locators/utils/copyLocatorToClipboard.ts:14]: It looks like magic strings to me now. 
Is it possible for you to add an explanation or constants with meaningful names to the code explaining them?

Comment (MariiaNebesnova) [reply src/features/locators/utils/copyLocatorToClipboard.ts:14]: sorry, not my code. but it's covered with tests, hope they could help you to understand what's going on

Comment (MariiaNebesnova) [reply src/features/locators/utils/copyLocatorToClipboard.ts:14]: I just refactored location of this function

Comment (Iogsotot) [reply src/features/locators/utils/copyLocatorToClipboard.ts:14]: what tests cover this code?

**Review by kostrubin — APPROVED** [review]

File: src/features/locators/utils/copyLocatorToClipboard.ts (hunk 1 above)
Comment (MariiaNebesnova) [reply src/features/locators/utils/copyLocatorToClipboard.ts:14]: `"escape symbols in locator"` from https://github.com/jdi-testing/jdn-ai/pull/1439/files#diff-c3f66e9470dbf46aa684c5ffec7394f39ed3ff49ba5e11d6aa1427f317a57938

**Pull request merged**